packages/headless/src/controllers/commerce/facets/core/headless-core-commerce-facet.ts

```diff
@@ -125,14 +156,34 @@ export function buildCoreFacet(
   return {
     ...controller,
 
-    toggleSelect: (selection: FacetValue) =>
-      dispatch(toggleSelectFacetValue({facetId, selection})),
+    toggleSelect: (selection: AnyFacetValue) => {
+      if (props.options.toggleSelectActionCreator) {
+        dispatch(props.options.toggleSelectActionCreator({selection, facetId}));
+        if (props.options.fetchResultsActionCreator) {
+          dispatch(props.options.fetchResultsActionCreator());
+          // TODO: analytics
```

Comment:
I got rid of analytics for facets altogether for the moment.

I think it's a waste of time to dispatch the old client-side events if we're never going to support them in the new commerce sub-package, and we need the new stack for the NRF demo anyways.